cmd/gitbase/server.go

```diff
@@ -25,12 +24,12 @@ var enableUnstableSquash = os.Getenv("UNSTABLE_SQUASH_ENABLE") != ""
 type cmdServer struct {
 	Verbose bool `short:"v" description:"Activates the verbose mode"`
 
-	Git      string `short:"g" long:"git" description:"Path where the git repositories are located"`
-	Siva     string `long:"siva" description:"Path where the siva repositories are located"`
-	Host     string `short:"h" long:"host" default:"localhost" description:"Host where the server is going to listen"`
-	Port     int    `short:"p" long:"port" default:"3306" description:"Port where the server is going to listen"`
-	User     string `short:"u" long:"user" default:"root" description:"User name used for connection"`
-	Password string `short:"P" long:"password" default:"" description:"Password used for connection"`
+	Git      []string `short:"g" long:"git" description:"Path where the git repositories are located, multiple directories can be defined"`
+	Siva     []string `long:"siva" description:"Path where the siva repositories are located, multiple directories can be defined"`
```

Comment:
Don't we want a short flag for siva files?

Reply:
I am not changing this, was defined as it is now.